modules/websession/lib/webuser.py

```diff
@@ -162,6 +162,9 @@ def page_not_authorized(req, referer='', uid='', text='', navtrail='', ln=CFG_SI
     elif CFG_ACCESS_CONTROL_LEVEL_SITE == 2:
         title = CFG_WEBACCESS_MSGS[6]
         body = "%s %s" % (CFG_WEBACCESS_MSGS[4], CFG_WEBACCESS_MSGS[2])
+    else:
+        title = CFG_WEBACCESS_MSGS[7]
+        body = "%s %s" % (CFG_WEBACCESS_MSGS[4], CFG_WEBACCESS_MSGS[2])
```

Comment:
Mmh `CFG_WEBACCESS_MSGS[7]` is *This functionality is temporarily closed due to server maintenance. Please use only the search engine in the meantime.* which is a bit long for a title, while `CFG_WEBACCESS_MSGS[4]` corresponds to *The site is temporarily closed for maintenance.  Please come back soon.*

What about using still `CFG_WEBACCESS_MSGS[6]` for the title and `CFG_WEBACCESS_MSGS[7]` for the first part of the body?